scripts/runDevInstance.sh

```diff
@@ -2,5 +2,6 @@
 
 # We use node directly rather than going through yarn so that we can use --inspect-publish-uid=http,
 # which silences most of the annoying debugger console logs on start.
-vercel env pull .env.local --environment=${1:-development} 2> /dev/null && \
+#vercel env pull .env.local --environment=${1:-development} && \
```

Comment:
I don't think this was meant to be included

Reply:
Oops, reverting